### PR TITLE
Fabric channel

### PR DESCRIPTION
**Pull request by JessicaXYWang**
Temporarily disable the generation of certain documents while waiting for updates to the notebook.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [notebooks/fabric_template/classification-before-and-after-synapseml-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Fclassification-before-and-after-synapseml-end.md) | 85 | 100 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fc65698f-04f3-4bce-8657-8b24194cbd15) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/cognitive-services-create-a-multilingual-search-engine-from-forms-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Fcognitive-services-create-a-multilingual-search-engine-from-forms-end.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46197f59-9e03-4cff-845c-d5f6b911b212) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/cognitive-services-multivariate-anomaly-detection-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Fcognitive-services-multivariate-anomaly-detection-end.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/86591ca9-9ba0-4b3b-9769-343ffd4f17dd) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/conditional-knn-exploring-art-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Fconditional-knn-exploring-art-end.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/381a869a-7531-4a4b-800b-d597a37189c0) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/hyperparameter-tuning-fighting-breast-cancer-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Fhyperparameter-tuning-fighting-breast-cancer-end.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fdf937c8-52e7-4ec4-98ea-29e99b369a29) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/isolation-forest-multivariate-anomaly-detection-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Fisolation-forest-multivariate-anomaly-detection-end.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e72bbb99-ee6f-4140-a993-0b18343f52ba) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/lightgbm-overview-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Flightgbm-overview-end.md) | 85 | 100 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/537350a3-706a-4abc-bad1-fc218fd88eb9) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/onnx-overview-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Fonnx-overview-end.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/55cfb6c9-069c-417d-8ca5-e5128becaddf) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/overview-cognitive-services-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Foverview-cognitive-services-end.md) | 85 | 100 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/38a9b491-3da1-4ad6-bd35-a58a3a584ef9) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/synapseml-first-model-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Fsynapseml-first-model-end.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/45df4e56-bae8-41c6-a741-64649cca1a6f) | [:white_check_mark:](# "Successfully processed") |
| [notebooks/fabric_template/tabular-shap-explainer-end.md](https://github.com/microsoft/SynapseML/blob/1eeb639a19ffced69a6284b01e528c958a24476e/notebooks%2Ffabric_template%2Ftabular-shap-explainer-end.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4face0e0-b321-4346-ac85-41e1c27ab00d) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by BrendanWalsh** [comment]
@mhamilton723 is traveling at the moment and can't comment himself, but he wanted to suggest finding a way to fold the "next steps" tabs into the notebooks themselves

File: docs/manifest.yaml [hunk 1]
```diff
@@ -1,6 +1,186 @@
...
+          title: Build a Search Engine
+          description: Build a Search Engine
+          ms.topic: overview
+          ms.reviewer: jessiwang
+          author: JessicaXYWang
+          ms.author: jessiwang
+      - path: ../../notebooks/features/cognitive_services/CognitiveServices - Multivariate Anomaly Detection.ipynb
+        filename: cognitive-services-multivariate-anomaly-detection
+        is_active: false # clear
+        footer: ../../notebooks/fabric_template/cognitive-services-multivariate-anomaly-detection-end.md
```
Comment (mhamilton723) [inline docs/manifest.yaml:35]: i dont think ../../notebooks/fabrictemplates is in this PR

**Review by mhamilton723 — COMMENTED** [review]
I think you might be missing some files in thgis PR

File: docs/python/documentprojection/channels/fabric.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+from ..utils.logging import get_log
+
+log = get_log(__name__)
+
+class FabricDoc(Document):
+    def __init__(self, content, metadata):
+        self.content = content
+        self.metadata = metadata
+
+class FabricPublisher(Publisher):
```
[the remark this answers is not in the document]
Comment (JessicaXYWang) [reply docs/python/documentprojection/channels/fabric.py:28]: thanks. compare_doc function and testing option in pipeline added to compare generated doc with a given md. file

**Comment by JessicaXYWang** [comment]
> I think you might be missing some files in thgis PR

Thanks. the files are no longer required. removed all reference to the files.

**Pull request closed without merging**